Source/WebCore/platform/mac/ScrollbarsControllerMac.mm

```diff
@@ -957,8 +958,7 @@ - (void)invalidate
         auto newVerticalPainter = retainPtr([NSScrollerImp scrollerImpWithStyle:newStyle controlSize:(NSControlSize)verticalScrollbar->widthStyle() horizontal:NO replacingScrollerImp:oldVerticalPainter]);
 
         [m_scrollerImpPair setVerticalScrollerImp:newVerticalPainter.get()];
-        macTheme->setNewPainterForScrollbar(*verticalScrollbar, WTFMove(newVerticalPainter));
-        macTheme->didCreateScrollerImp(*verticalScrollbar);
+        static_cast<ScrollbarMac*>(verticalScrollbar)->setNewPainterForScrollbar(WTFMove(newVerticalPainter));
```

Comment:
Would feel a bit nicer if we could `dynamicDowncast<>`

Reply:
Added isMacScrollbar so this is possible.